### ISSUE TITLE
Conflict with plugin tray and local graph onclick

### ISSUE THREAD

**Issue by EduardDE7** [nekoshita/obsidian-auto-card-link#63]
When clicking the link closes Obsidian in tray and the local graph stops working

**Comment by gratzel** [comment]
Same problem.  When I enable the Tray plugin and click on an Auto Card, Obsidian minimizes instead of opening the link.

**Comment by ormai** [comment]
Does it also happen with a normal external link?

**Comment by gratzel** [comment]
No, normal external links still work correctly.  If I disable the Tray plugin, the Auto Card links work.

If there’s anything I can troubleshoot, let me know.  No error is logged in the chrome console.  I tried to examine the Javascript but I don’t know enough to follow how and where this behavior is handled.

**Comment by ormai** [comment]
I was able to reproduce the problem following these steps:
1. install the 'Tray' community plugin
2. enable 'Run in background' from the plugin settings
3. click on any cardlink.

Side note: on my system (Gnome 48 on Arch Linux) Obsidian was hidden, not minimized, and there was no way of bringing it back in the foreground (Gnome has no tray). I had to manually terminate all the Electron processes and restart Obsidian.

Since a cardlink is just an anchor element it shouldn't behave any differently from a normal external link. So I checked for differences between the HTML attributes of `a.external-link` and the ones of `a.auto-card-link-card`.

Adding `target="_blank"` to `a.auto-card-link-card` solves the issue, tested in the web dev tools.

This is an extremely simple fix but I'm afraid it wouldn't have any luck getting merged soon 🫤.
